src/bind.py

```diff
@@ -266,24 +266,44 @@ def _install_snap_package(
             logger.exception(error_msg)
             raise InstallError(error_msg) from e
 
-    def _zones_to_files_content(self, zones: list[models.Zone]) -> dict[str, str]:
+    def _zones_to_files_content(
+        self, zones: list[models.Zone], topology: models.Topology
+    ) -> dict[str, str]:
         """Return zone files and their content.
 
         Args:
             zones: list of the zones to transform to text
+            topology: Topology of the current deployment
 
         Returns:
             A dict whose keys are the domain of each zone
             and the values the content of the zone file
         """
         zone_files: dict[str, str] = {}
         for zone in zones:
-            content = constants.ZONE_HEADER_TEMPLATE.format(
+            content = constants.ZONE_APEX_TEMPLATE.format(
```

Comment:
Generally speaking it is better to use f-strings that format strings because:
* You have to manually match the variable names in the format strings to arguments to the format function
* There is indirection in that the format string is defined somewhere else compared to where it is used
* It is easy to visualise what the final value of the f-string will look like as all the computation is done exactly where the value is inserted

Format strings can be worth it if you have the same template used over and over. In this case it seems that the format string is only used once so it would probably be more readable to in-line them

Reply:
I understand and it makes sense to me.  
At the same time, using format strings for those allow me to regroup their definitions in `src/constants.py`. I could even create a file later regrouping all format strings if need be. That way, I remove the templates from the code and have a central place to check for their definitions.

Reply:
Ok, perhaps in that case change the name of the central file `zone_templates` or something to indicate that is where all the DNS templates are defined. Calling it constants implies that they are re-used across the code base

Reply:
I've created a new file called "templates.py" and moved all the templates there